cmd/server.go

```diff
@@ -41,13 +43,35 @@ func init() {
 	echox.MustViperFlags(viper.GetViper(), serveCmd.Flags(), APIDefaultListen)
 	echojwtx.MustViperFlags(viper.GetViper(), serveCmd.Flags())
 
-	// audit log path
-	serveCmd.Flags().String("audit-log-path", "/app-audit/audit.log", "Path to the audit log file")
-	viperx.MustBindFlag(viper.GetViper(), "audit.log.path", serveCmd.Flags().Lookup("audit-log-path"))
+	// only available as a CLI arg because it shouldn't be something that could accidentially end up in a config file or env var
+	serveCmd.Flags().BoolVar(&serveDevMode, "dev", false, "dev mode: enables playground, disables all auth checks, sets CORS to allow all, pretty logging, etc.")
+	serveCmd.Flags().BoolVar(&enablePlayground, "playground", false, "enable the graph playground")
 }
 
 func serve(ctx context.Context) {
-	err := otelx.InitTracer(config.AppConfig.Tracing, appName, logger.Sugar())
+	if serveDevMode {
+		enablePlayground = true
+		config.AppConfig.Logging.Debug = true
+		config.AppConfig.Logging.Pretty = true
+		config.AppConfig.Server.WithMiddleware(middleware.CORS())
+		// this is a hack, echojwt needs to be updated to go into AppConfig
+		viper.Set("oidc.enabled", false)
+	}
+
+	// natsClient, err := configureNatsClient()
```

Comment:
We want to add this / something like this back in to generate our `pubsub.Client` so that we can add it to our entclient right?